sample/src_polymer/elements/scene-perception-pages/main.js

```diff
@@ -0,0 +1,301 @@
+const MIN_ACCEPTABLE_QUALITY = 0.25;
```

Comment:
Maybe we'd better use a closure to hide all the variables/functions from `document`? Only export necessary ones.